composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "require-dev": {
+        "phpunit/phpunit": "^9.6"
+    },
+    "require": {
+        "silverstripe/supported-modules": "dev-pulls/main/combine-all-versions"
```

Comment:
```suggestion
        "silverstripe/supported-modules": "dev-main"
```

Reply:
Yup, that'll be done once the PR for that fork is merged, so we can easily rerun CI here against that PR to see it go green if changes are needed over there.

Reply:
Done